packages/buefy-next/src/components/skeleton/Skeleton.vue

```diff
@@ -1,77 +1,76 @@
 <script>
-import { h as createElement } from 'vue'
+import { createCommentVNode, h as createElement } from 'vue'
 
-export default {
-    name: 'BSkeleton',
-    functional: true,
-    props: {
-        active: {
-            type: Boolean,
-            default: true
-        },
-        animated: {
-            type: Boolean,
-            default: true
-        },
-        width: [Number, String],
-        height: [Number, String],
-        circle: Boolean,
-        rounded: {
-            type: Boolean,
-            default: true
-        },
-        count: {
-            type: Number,
-            default: 1
-        },
-        position: {
-            type: String,
-            default: '',
-            validator(value) {
-                return [
-                    '',
-                    'is-centered',
-                    'is-right'
-                ].indexOf(value) > -1
+const BSkeleton = (props) => {
+    if (!props.active) return createCommentVNode()
```

Comment:
@hirokiky Could you explain why this change is necessary? The only description I found in [Vue 3's documentation](https://vuejs.org/guide/extras/render-function.html#functional-components) says
> The signature of a functional component is the same as the setup() hook:

But I could not find any official statement that we should not return `undefined` from a functional component. If I changed the line to return `undefined`, it neither crashed nor warned with Vue 3.4.19. If I viewed on the inspector, it was replaced with a comment `<!---->`

Your change works anyway. This is just out of curiosity, for an educational purpose.

Reply:
Thank you for checking the behavior.

I thought it should return VNode, but there is normalize function to convert undefined/null into CommentNode.
https://github.com/vuejs/core/blob/main/packages/runtime-core/src/vnode.ts#L747

so we don't need to return it explicitly.

Reply:
> so we don't need to return it explicitly.

We will keep your commit anyway.

Reply:
I have overlooked your latest commit. Thanks for updating it!